flow-typed/submissions.js

```diff
@@ -178,3 +178,14 @@ declare type GeneratePaymentConfigPayload = {
   crn2?: string,
   crn3?: string,
 }
+
+declare type NewFormSubmissionFileAccessToken = {
+  expiry: string,
```

Comment:
can we rename this to `expiresAt` to match the standard we used for other timestamps